app/presenters/tree_builder_alert_profile_obj.rb

```diff
@@ -0,0 +1,56 @@
+class TreeBuilderAlertProfileObj < TreeBuilder
+  def initialize(name, type, sandbox, build = true, assign_to: nil, cat: nil, objects: nil)
+    @assign_to = assign_to
+    @cat = cat
+    @objects = objects
+    @cat_tree = @assign_to.ends_with?("-tags")
+    super(name, type, sandbox, build)
+  end
+
+  def override(node, object, _pid, _options)
+    node[:title] = (object.name.presence || object.description) unless object.kind_of?(MiddlewareServer)
+    node[:hideCheckbox] = false
+    node[:select] = @objects.include?(object.id)
+    node
```

Comment:
Lame :-D I did not notice it because I blindly refactored this from the previous version where node was passed as instance variable :-)